src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -476,6 +476,11 @@ public override AbstractPager OpenJournalPager(long journalNumber)
                 var path = Path.Combine(_journalPath, name);
                 if (File.Exists(path) == false)
                     throw new InvalidOperationException("No such journal " + path);
+                if (new FileInfo(path).Length == 0)
```

Comment:
No, need to do this properly.
Also, need to handle errors here, what if we cannot write (common in recovery scenario).